cpp/tests/utility/Eigen.cpp

```diff
@@ -184,9 +184,7 @@ TEST(Eigen, ComputeJTJandJTr) {
     ref_JTr << 0.477778, -0.262092, -0.162745, -0.545752, -0.643791, -0.883007;
 
     auto testFunction = [&](int i, Eigen::Vector6d &J_r, double &r) {
-#ifdef _OPENMP
 #pragma omp critical
```

Comment:
Seems like this `#pragma omp critical` can be removed. Other lambda functions used in `utility::ComputeJTJandJTr` do not have this pragma surrounded. It works for me on Ubuntu. Let's try to see if it can pass the CI.

Reply:
Agree, will do